packages/core/test/integration/model/findAll.test.js

```diff
@@ -26,7 +26,9 @@ describe(Support.getTestDialectTeaser('Model'), () => {
       intVal: DataTypes.INTEGER,
       theDate: DataTypes.DATE,
       aBool: DataTypes.BOOLEAN,
-      binary: DataTypes.BLOB,
+      ...(dialectName === 'oracle'
+        ? { binary: DataTypes.STRING(16, true) }
+        : { binary: DataTypes.BLOB }),
```

Comment:
Hopefully I remind myself to check this out again later. I feel like we might want to make `BLOB` an alias for `STRING(16, true)`, but I need to think more about it